app/src/pages/CollectionPage/CollectionPage.tsx

```diff
@@ -204,15 +224,15 @@ export const CollectionPage = (): React.ReactElement => {
                   <Stack direction={Direction.Horizontal} childAlignment={Alignment.Center} contentAlignment={Alignment.Center} shouldAddGutters={true} defaultGutter={PaddingSize.Wide}>
                     <MetricView name={'Items'} value={`${collectionStatistics.itemCount}`} />
                     <MetricView name={'Owners'} value={`${collectionStatistics.holderCount}`} />
-                    <MetricView name={'Total Volume'} value={`${collectionStatistics.totalTradeVolume}`} />
+                    <MetricView name={'Total Volume'} value={`${ethers.utils.formatEther(collectionStatistics.totalTradeVolume)}`} />
                   </Stack>
                   <ResponsiveHidingView hiddenBelow={ScreenSize.Medium}>
                     <Box variant='divider' isFullHeight={true} width='1px' />
                   </ResponsiveHidingView>
                   <Stack direction={Direction.Horizontal} childAlignment={Alignment.Center} contentAlignment={Alignment.Center} shouldAddGutters={true} defaultGutter={PaddingSize.Wide}>
-                    <MetricView name={'24h Low Sale'} value={`Ξ ${collectionStatistics.totalTradeVolume}`} />
-                    <MetricView name={'24h High Sale'} value={`Ξ ${collectionStatistics.highestSaleLast24Hours}`} />
-                    <MetricView name={'24h Volume'} value={`Ξ ${collectionStatistics.tradeVolume24Hours}`} />
+                    <MetricView name={'24h Low Sale'} value={`Ξ ${ethers.utils.formatEther(collectionStatistics.lowestSaleLast24Hours)}`} />
+                    <MetricView name={'24h High Sale'} value={`Ξ ${ethers.utils.formatEther(collectionStatistics.highestSaleLast24Hours)}`} />
+                    <MetricView name={'24h Volume'} value={`Ξ ${ethers.utils.formatEther(collectionStatistics.tradeVolume24Hours)}`} />
```

Comment:
The collection statistics display for some collections and it doesn't for some. for example;  meebit collection stats it throws this error : 
![image (4)](https://user-images.githubusercontent.com/37664545/169274686-7f6a6334-29ae-4e07-8c3c-2c5d7087c953.png) 
the traded volume in 24hrs coming from the API is: `8.464904709889e+23`
.